api/rest/v1/index.md

```diff
@@ -88,3 +88,11 @@ Methods for retrieving statistical variable related data.
 | ----------------------------------- | ------------------------------------------------- | --------------------------------------------------- |
 | Variables                           | [/v1/variables](/api/rest/v1/variables)           | Get all variables associated with a specific entity |
 | Variables <bulk-tag>bulk</bulk-tag> | [/v1/bulk/variables](/api/rest/v1/bulk/variables) | Get all variables available for multiple entities   |
+
+### Other Query Languages
```

Comment:
Maybe " ### Graph Query" and the description below could be specific to SPARQL

Reply:
Changed header to 'Graph Query' and make description specific to SPARQL